src/ftest.jl

```diff
@@ -77,16 +84,21 @@ function ftest(mods::LinPredModel...)
     df2 = Float64.(df2)
     prepend!(df2, [NaN])
 
-    Δdf1 = diff(df1)
-    prepend!(Δdf1, [NaN])
-
     R2 = collect(r2.(mods))
-    ΔR2 = diff(R2)
+    ΔR2 = -diff(R2)
     prepend!(ΔR2, [NaN])
 
-    ΔSSR = diff(SSR)
+    ΔSSR = -diff(SSR)
     prepend!(ΔSSR, [NaN])
-    return CoefTable([df1, Δdf1, nparams, -df2, SSR, ΔSSR, R2, ΔR2, fstat, pval],
-                     ["Residual DOF", "ΔResidual DOF", "DOF", "Δdof", "SSR", "ΔSSR", "R²", "ΔR²", "F*", "p(>F)"],
+    return CoefTable([df1, nparams, df2, SSR, ΔSSR, R2, ΔR2, fstat, pval],
+                     ["Res. DOF",
```

Comment:
OTOH, one name per line is a bit too much. Just break the line into two (or three) parts somewhere before 92 chars.